src/hypothesis/internal/compat.py

```diff
@@ -392,7 +392,7 @@ def __add__(self, value):
         return compatbytes(bytearray.__add__(self, value))
 
     def __radd__(self, value):
-        return compatbytes(bytearray.__radd__(self, value))
+        return compatbytes(bytearray.__add__(value, self))
```

Comment:
This was actually just a bug that I ran into. It turns out bytearray.__radd__ doesn't exist, but this never actually got invoked in normal code (it got invoked in my code because I made a mistake and I fixed this to try to get a better error).

Properly I should add a test for this, but actually it might be better to just delete this method.